overdrive.py

```diff
@@ -247,15 +251,18 @@ def _get_book_list_page(self, link, rel_to_follow='next'):
         """
         # We don't cache this because it changes constantly.
         status_code, headers, content = self.get(link, {})
-        data = json.loads(content)
+        if isinstance(content, basestring):
+            content = json.loads(content)
```

Comment:
See https://github.com/NYPL-Simplified/circulation/pull/455/files#r103262011. I made this change, and similar changes in core, to accommodate the way Overdrive is mocked in the circulation tests.